inference-engine/src/transformations/include/transformations/smart_reshape/proposal_scales_stridedslice.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API opset1_ProposalScales;
```

Comment:
I think Proposal1Scales and Proposal4Scales look more natural regarding other transformation names